docs/javascript/browser-apis-and-jquery.md

```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
 
 var catIntervalID = setInterval(increaseCatLikes, 5000);
 ```
+
+### jQuery Events
+
+The following `event methods` handle events on our webpage.
+
+The `click()` method allows us to execute a function when element is clicked.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Events!</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+    <button id="name">Click</button>
+    <h1 id="heading">Welcome!</h1>
+    <span class="material-icons-outlined" id="wave">
+        waving_hand
+        </span>
+    <form>
+        <input type="text" placeholder="Enter Wizard Name" id="wizardName" autocomplete="off">
+        <input id="sub" type="submit" value="Submit">
+    </form>
+
+
+    <script>
+
+    function clickHeading() {
+        $("#heading").show();
+    }
+    // when button is clicked it will show the hidden heading text
+    $("#name").click(clickHeading);
+
+
+    </script>
+
+</div>
+
+</html>
+```
+
+The `dblclick()` method allows us to execute a function when element is double clicked.
```

Comment:
This wasn't in the list of methods to add docs for, and I don't see it in any current courses when I `grep` around the CWHQ projects repo.